ui/app/AppLayouts/Profile/helpers/ProfileShowcaseSettingsModelAdapter.qml

```diff
@@ -59,6 +59,10 @@ QObject {
                 expectedRoles: ["members"]
             }
         ]
+        filters: ValueFilter {
+            roleName: "joined"
```

Comment:
should owned be included communities here too?

Reply:
joined is false on owned communities?

Reply:
in the video above the owned communities are not filtered. Is there a corner case I'm missing? 🤔

Reply:
I'm testing it and the ones that the user is the `Owner` are correctly shown!